glue/core/roi.py

```diff
@@ -163,21 +177,37 @@ class RectangularROI(Roi):
 
     """
     A 2D rectangular region of interest.
+
+    Parameters
+    ----------
+    xmin, xmax :  float, optional
+        x coordinates of left and right border
+    ymin, ymax :  float, optional
+        y coordinates of lower and upper border
+    theta : float, optional
+        Angle of anticlockwise rotation around center
     """
 
-    def __init__(self, xmin=None, xmax=None, ymin=None, ymax=None):
+    def __init__(self, xmin=None, xmax=None, ymin=None, ymax=None, theta=None):
         super(RectangularROI, self).__init__()
         self.xmin = xmin
         self.xmax = xmax
         self.ymin = ymin
         self.ymax = ymax
+        self.theta = 0 if theta is None else theta
+        if np.isclose(self.theta % np.pi, 0.0, atol=1e-9):
+            self.rotation = np.identity(2)
```

Comment:
In general I think it would be good to use ``rotation_matrix`` instead of ``rotation`` to make it clear it's a matrix and not an angle.